frontend/test/metabase/scenarios/collections/collections-sidebar.cy.spec.js

```diff
@@ -0,0 +1,47 @@
+import { restore } from "__support__/e2e/cypress";
+
+describe("collections sidebar (metabase#15006)", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+
+    cy.visit("/collection/root");
+  });
+
+  describe("in desktop form factors", () => {
+    it("should display collections sidebar on page load", () => {
+      cy.findByText("First collection").should("be.visible");
+    });
+
+    it("should not be able to close collections sidebar from sidebar itself", () => {
+      cy.icon("close").should("not.be.visible");
+    });
+
+    it("should not be able to toggle collections sidebar from burger icon", () => {
+      cy.icon("burger").should("not.be.visible");
+    });
```

Comment:
https://docs.cypress.io/guides/references/best-practices#Creating-tiny-tests-with-a-single-assertion

> Anti-Pattern: Acting like you're writing unit tests.

> Best Practice: Add multiple assertions and don't worry about it

Reply:
Just this block split into tiny chunks:
![image](https://user-images.githubusercontent.com/31325167/124741693-4551d000-df1c-11eb-8969-0d57f35d6cac.png)

Single test only
![image](https://user-images.githubusercontent.com/31325167/124741952-85b14e00-df1c-11eb-8f57-0bda8f70fa7a.png)